### ISSUE TITLE
improve phrasing about a policy that can block new Teams client deployment

### ISSUE THREAD

**Issue by orandev** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#9924]
Hello,
We have a "Restrict sign-in to other tenants" policy on our computers and this blocks me from seeing the "Try the new Teams" toggle switch.
But I didn't understand from the documentation that this policy would block the new Teams client.
Could you rephrase it like that please?

If the user is on a VDI computer (Citrix, VMware etc.).
If the user is signed in to classic Teams with a Teams for Life account and a work account.
If the user is signed in to classic Teams with a Teams for Life account.
If you have an MSIX client.
**If there is a "Restrict sign in to Teams to accounts in specific tenants" policy in place on the user's computer (Learn more about how to restrict Teams sign in on desktop devices)**

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1d50c2e0-d6d3-ec3f-5e99-acaa2a6868d5
* Version Independent ID: 1d50c2e0-d6d3-ec3f-5e99-acaa2a6868d5
* Content: [The new Microsoft Teams desktop client - Microsoft Teams](https://learn.microsoft.com/en-us/microsoftteams/new-teams-desktop-admin?tabs=teams-admin-center)
* Content Source: [Teams/new-teams-desktop-admin.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/new-teams-desktop-admin.md)
* Service: **msteams**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by scanum** [comment]
Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by get-itips** [comment]
This has been added in https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/commit/098f4100726f577682ca87020cbe0467785adc9e

thanks for your feedback
